client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -51,12 +50,24 @@
 		}
 	}
 
+	.design-selector__design-option {
+		transition: transform 0.3s ease-out, box-shadow 0.3s ease-out;
```

Comment:
We've generally been using `spring` to make sure animations are more smooth and natural feeling. In this specific case I don't see much difference tho. @sirreal any thoughts?

Reply:
I see 8K difference in the bundle size 😄

Reply:
I think we should be using `react-spring` for everything: p1576859343088200-slack-luna

It's fairly clear to me from that comment that we'll be using it. Gutenenberg is also using it and that will lend to the consistent feel

I'd say we should be using it _more_ extensively (all transitions?) and amortize that bundle size cose.

Reply:
Right now yes but we're going to have spring animations elsewhere in the flow (in style picker step) so those kb's are bound to come back. :-)

Reply:
Is there a case for using spring now and in this particular instance? 🤔 

Totally understand if we want more complex animation effects, but given how much crud we just rid ourselves of for the sack of a hover effect...

Reply:
>now and in this particular instance? 

Yeah, put that way, I'd be fine removing this time. 👍 😃

Reply:
I disagree with removing it at this time. I see many places we should be adding spring animations and I believe we'll want to transition back to react-spring

> as it provides more natural and consistent motion.

I expect to use it for:

- themed vertical backgrounds
- transitions between steps

I will _disagree and commit_ if folks decide to remove it here.

Reply:
> I disagree with removing it at this time.

Do you mean removing it all together? Or removing it for the hover+scale effect of the design thumbnails for this component until we need it later? 

Agree with all your points. I just mean the latter. 

I'm not invested in either approach, so I'd be willing to follow your guidance.

Reply:
After recent conversation, the Spring conversation has sprung.

_Rip it out!_ 👍